robot/Cumulus/resources/locators_51.py

```diff
@@ -335,7 +335,11 @@
         'temp_builder_labels':'//lightning-input[@class="slds-truncate slds-form-element"]//label',
         'ge_form_labels':'//label[@class="slds-form-element__label" or @class="slds-form-element__label slds-no-flex"]',
         'temp_builder_sections':'//h2[@class="slds-card__header-title"]',
-        'ge_form_sections':'//span[contains(@title,"{}")]'
+        'ge_form_sections':'//span[contains(@title,"{}")]',
+        'batch_checkbox':'//*[contains(@data-qa-locator, "Require")]//child::input',
```

Comment:
please use the locator in line 308 instead of creating this

Reply:
fixed it, thanks.